src/test/kotlin/org/rust/cargo/RustWithToolchainTestBase.kt

```diff
@@ -15,7 +15,7 @@ import org.rust.cargo.project.settings.rustSettings
 import org.rust.cargo.toolchain.RustToolchain
 
 // This class allows to execute real Cargo during the tests.
-// Unlike `RustTestCaseBase` it does not use in-memory temporary VFS
+// Unlike `RsTestBase` it does not use in-memory temporary VFS
 // and instead copies real files.
 abstract class RustWithToolchainTestBase : CodeInsightFixtureTestCase<ModuleFixtureBuilder<*>>() {
```

Comment:
Would it be useful to also rename `RustWithToolchainTestBase` -> `RsWithToolchainTestBase` to have similar naming?